docs/topics/email.txt

```diff
@@ -12,10 +12,10 @@ development, and to provide support for platforms that can't use SMTP.
 
 The code lives in the ``django.core.mail`` module.
 
-Quick example
-=============
+Quick examples
+==============
 
-In two lines::
+Use ``send_mail()`` for simpler cases. For example, to send a plain text only message::
```

Comment:
```suggestion
Use :func:`send_mail` for simpler cases. For example, to send a plain text only message::
```

(Using `:func:` will link to the documentation below. Sphinx automatically adds `()` in func links.)